README.md

```diff
@@ -67,6 +67,12 @@ cannot access native TCP sockets, the standard use of SSH over TCP is not possib
 some other stream transport like a websocket may be used. For details about the
 TypeScript library, see [src/ts/ssh/README.md](./src/ts/ssh/README.md).
 
+### Limitations
```

Comment:
This section should go in [`src/ts/ssh/README.md`](https://github.com/microsoft/dev-tunnels-ssh/blob/main/src/ts/ssh/README.md), as I think it's too minor an issue to surface in the top-level project readme.